script/script.js

```diff
@@ -72,22 +71,35 @@ const drawSquare = (position) => {
 };
 
 const mouseDownListener = (event) => {
```

Comment:
NIT: I'd call event listeners as handlers. E.g. `mouseDownHandler`. It's more common naming :)